doc/api/util.md

```diff
@@ -946,6 +946,161 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
+## `util.parseArgs([argv][, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `argv` {string[]|Object} (Optional) Array of argument strings; defaults
+  to [`process.argv.slice(2)`](process_argv). If an Object, the default is used,
+  and this parameter is considered to be the `options` parameter.
+* `options` {Object} (Optional) The `options` parameter is an
```

Comment:
The word "option" has three different meanings in the added documentation: (1) JavaScript options passed via an options object, (2) arguments with a value, (3) any argument that starts with a `-`. I can see this leading to confusion.

Reply:
Certainly we say "option" a lot, but I am unsure how to reconcile this given the word "option" is used in both JS and CLI contexts to mean entirely separate things.  Even if we adopted the IEEE standard terminology, we'd still have that problem.

Reply:
Why not be more verbose and say "command line parameter(s)"?

Reply:
I don't think there's any clear & obvious change to be made here.  if someone thinks of one, a future PR can address it, but IMO this is good enough for now